.reuse/dep5

```diff
@@ -0,0 +1,31 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
What reads this file? Trying to understand the value in maintaining it too.

Reply:
[REUSE](https://reuse.software/) can read this file, as can a lot of license evaluation tools that can parse the Debian copyright format (e.g. Fossology and others). Having the detailed copyright and license data for the whole project is beneficial for the users of the library who need to comply with the licenses.

Maintaining this file can be helped by e.g. Debian who will be able to pull and reuse it and later contribute updates back. Alternatively, keeping it up-to-date by the project itself can give you a good overview of the licensing situation.

(I’m a Debian Developer so I can help with the Debian bits of the above.)

Reply:
Hm OK. I doubt the project will change much at all, so there isn't much worry about maintaining the updates. It's yet another piece of metadata, but, OK if it has some value to debian packaging. I'd merge this change without the others.

Reply:
Okay. I still think the other changes are useful, but it’s up to you, of course, to not include them if you choose so.

I will resubmit the PR without the other commits.